app/views.py

```diff
@@ -1,30 +1,36 @@
 import json
-from flask import jsonify, make_response, request
+from flask import jsonify, request
 from dateutil.parser import parse
 from app import app
 from .punch import Punch
-from .helpers import get_topics_response, get_articles_response, get_articles_by_date, response
+from .helpers import get_topics_response, get_articles_response, response, get_articles_by_date_response
 
 punch = Punch()
 
 @app.route('/topics', methods=['GET'])
 def get_topics():
-    return get_topics_response(punch.getTopics(), 200)
+    return get_topics_response(punch.get_topics(), 200)
 
 @app.route('/articles/', methods=['GET'])
 def get_articles():
-    topic = request.args.get('topic').capitalize()
+    """
+    Get all the articles from Punch
+    """
+    topic = request.args.get('topic').lower()
     date_provided = request.args.get('date')
     try:
         str(topic)
         if date_provided:
             parse(date_provided).timestamp()
```

Comment:
For sanity, let's restrict the date to a specific format. The user should only be able to pass in the date in one format, preferably in iso-8601 date notation yyyy-mm-dd eg 2018-04-12.

You can also reassign date_provided here so you don't have to call `parse` again as you are down the line.

Reply:
I want flexibility so a user can enter any date format